### PR TITLE
docs: add connect to testnet instructions

### PR DESCRIPTION
**Pull request by hal3e**
closes: https://github.com/FuelLabs/fuels-rs/issues/553

### CODE REVIEW

File: docs/src/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Rust SDK for Fuel. It can be used for a variety of things, including but not limited to:
 
 - Compiling, deploying, and testing [Sway](https://github.com/FuelLabs/sway) contracts;
-- Launching a local Fuel node;
+- Use the [Tesnet](../providers/external-node.md) or run a local Fuel node;
```
Comment (digorithm) [inline docs/src/README.md:6]:
```suggestion
- Use the [Testnet](../providers/external-node.md) or run a local Fuel node;
```

File: docs/src/getting-started/connecting.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ At a high level, you can use the Fuel Rust SDK to build Rust-based applications
 
 For this interaction to work, the SDK must be able to communicate with a `fuel-core` node; you have two options at your disposal:
 
-1. Use the SDK's native `launch_provider_and_get_wallet()` that runs a short-lived test Fuel node;
-2. Run a Fuel node outside your SDK code (using `fuel-core`) and instantiate a provider that points to that Fuel node's IP and port.
+1. Use the [Tesnet](../providers/external-node.md) or run a Fuel node (using `fuel-core`) and instantiate a provider that points to that node's IP and port.
```
Comment (digorithm) [inline docs/src/getting-started/connecting.md:7]:
```suggestion
1. Use the [Testnet](../providers/external-node.md) or run a Fuel node (using `fuel-core`) and instantiate a provider that points to that node's IP and port.
```

File: docs/src/providers/external-node.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+
+    // Create the wallet.
+    let _wallet = WalletUnlocked::new_random(Some(provider));
 ```
+
+> **Note:** Assets for the Tesnet node can be obtained from the faucet at
+>
+>[faucet-beta-1.fuel.network](https://faucet-beta-1.fuel.network)
+>
+> There is also a block explerer for the Tesnet at
```
Comment (digorithm) [inline docs/src/providers/external-node.md:27]:
```suggestion
> There is also a block explorer for the Testnet at
```

**Review by Br1ght0ne — APPROVED** [review]
LGTM

**Review by MujkicA — APPROVED** [review]

**Review by segfault-magnet — APPROVED** [review]

**Review by digorithm — APPROVED** [review]
Good stuff! :shipit:

**Pull request merged**